### DISCUSSION TITLE
一些使用上的建议和使用上的一些问题

### DISCUSSION THREAD

**Discussion by Mulsd** [MCSLTeam/MCSL2#105] [功能建议] [answered]
既然跳到讨论版了我就干脆不上issue了
-------
首先是主页可以做成可供选择的控件（我不会Python，不知道你懂我意思没有，即文本可供选择）。
我想要改至MCSLAPI使用，但是还得我一个个字打，这不好（

其次是创建服务器选择核心这边，下载完核心后会有一点让人迷惑：软件有没有自动选择到我下载的核心？事实上是没有，然后又有问题：刚刚下载的核心放哪了？找一圈才在设置里发现路径。这里可以在下载时提示本次下载储存在/XX/xxx/，或者在第一次下载时提供手动选择下载到哪里的选择功能（为什么总是询问选择框是黑的？）。如果手动选择核心文件，虽然有小通知提示，但是最好像JAVA那项一样在醒目处标识到手动选择的核心（文件名即可）。

还有一点，FastMirror似乎有延迟问题。我下载mohist核心适用，但是它提供的构建版本过于老旧（2022年构建），这点应该能通过改边API来改善，但是转回到第一点（
-------
以上是我在使用时遇到和想到的问题与建议。此段内容并不是对软件的责难与批评并且具有极高的主观色彩，也不是强迫软件要这样做，毕竟隔行如隔山，我的一个小意见可能是莫大的工程量。但是这些微调能够给予用户更好的体验，最好能靠这个方向改进。不改也行，就是麻烦点
-------
测试环境Win1122h2，生产环境win1022H2(应该)，暂未遇到致命性问题

**Comment by lxhtt** [comment]
1.可供选择的控件会考虑  
2.自动选择核心工程量过于庞大  
3.FastMirror以外，我们以后还会考虑对接更多的镜像站API，敬请期待。  
4.别太客气

**Comment by lxhtt** [comment]
主页公告将在下个版本中提供双击复制的功能

**Comment by lxhtt** [comment, marked as answer]
基本完成
